test/scrabble_test.rb

```diff
@@ -1,13 +1,33 @@
 gem 'minitest'
 require_relative '../lib/scrabble'
 require 'minitest/autorun'
-require 'minitest/emoji'
+require 'minitest/pride'
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_hello
+    assert_equal 8, Scrabble.new.score("hello")
+  end
+
+  def test_it_can_score_an_empty_string
+    binding.pry
+    assert_equal 0, Scrabble.new.score("")
+  end
+
+  def test_it_can_score_nil
+    assert_equal 0, Scrabble.new.score(nil)
+  end
+
+  def test_it_can_score_a_crazy_word
+    assert_equal 56, Scrabble.new.score("supercalifragilisticexpialidocious")
```

Comment:
It's intriguing to me that you may have perhaps known the word score for this word that I refuse to re-type. Alternatively, you took the time to score it for this test. Kudos either way.